src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -79,8 +99,21 @@ public static int Main(string[] args)
             app.Description = "Initializes empty project for .NET Platform";
             app.HelpOption("-h|--help");
 
+            var csharp = app.Option("-c#|--csharp", "Create an C# project", CommandOptionType.NoValue);
```

Comment:
@brthor, we could technically support both options. `dotnet new --lang fsharp` sounds pretty long to type so I'd still consider: `dotnet new fsharp` considering that this is first user experience. Also: should we default to csharp? If `dotnet new` displayed small usage help so you could copy&paste the command it wouldn't be too bad but:
`dotnet new --help` + `dotnet new --lang fsharp` does not sound too fun